client/src/screens/Analytics.js

```diff
@@ -14,20 +18,160 @@ export default class Analytics extends Component {
         setFacebookData(this.props);
     }
 
+    constructor(props) {
+        super(props);
+        this.toggle = this.toggle.bind(this);
+        this.select = this.select.bind(this);
+        this.state = {
+            dropdownOpen: false,
+            value : "Analytics View",
+            posts : [
+                {
+                    id: 1,
+                    topic: "Facebook",
+                    btnSelected: false
+                },
+                {
+                    id: 2,
+                    topic: "Instagram",
+                    btnSelected: false
+                },
+                {
+                    id: 3,
+                    topic: "Twitter",
+                    btnSelected: false
+                },
+                {
+                    id: 4,
+                    topic: "Overall",
+                    btnSelected: true
+                },
+            ]
+        };
+    }
+
+    toggle() {
+        this.setState({
+            dropdownOpen: !this.state.dropdownOpen
+        });
+    }
+
+    select(event) {
+        this.setState({
+            dropdownOpen: !this.state.dropdownOpen,
+            value: event.target.innerText
+        });
+    }
+
     render() {
-        return (
-            <div>
-                <Container>
-                    <Row>
-                        <Col>
-                            <InstagramLikesAndComments {...this.props} />
-                        </Col>
-                        <Col>
-                            <TwitterFollowerRatio {...this.props} />
-                        </Col>
-                    </Row>
-                </Container>
-            </div>
-        );
+
+        const { value } = this.state;
+
+        if(value == "Instagram"){
```

Comment:
Can you make `Instagram`, `Facebook`, and `Twitter` into variables that are just strings? Like we have our `actions`

Reply:
taken care of 👍